src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide(num) {
+      this.result /= num;
```

Comment:
There is a potential division by zero error in the `divide` method. You should add a check to ensure that `num` is not zero before performing the division to prevent runtime errors.